app/jobs/regular/user_email.rb

```diff
@@ -153,7 +153,7 @@ def message_for_email(user, post, type, notification, args = nil)
       raise Discourse::InvalidParameters.new("type=#{type}") unless UserNotifications.respond_to?(type)
 
       email_args[:email_token] = email_token if email_token.present?
-      email_args[:new_email] = user.email if type.to_s == "notify_old_email"
+      email_args[:new_email] = user.email if type.to_s == "notify_old_email" || type.to_s == "notify_old_email_add"
```

Comment:
What's the "type" of the `type` variable? Isn't it a symbol? If it is, when are we converting it to a string instead of checking directly against other symbols?

Reply:
Sidekiq converts symbol arguments to strings, so it should be a string, unless the job is executed directly.